Cargo.toml

```diff
@@ -143,7 +143,10 @@ proptest = "1.1.0"
 tempfile = "3.8"
 termcolor = "1.2"
 thiserror = "1.0"
-tokio = { version = "1.26.0", default-features = false, features = ["macros", "rt-multi-thread"] }
+tokio = { version = "1.26.0", default-features = false, features = [
```

Comment:
So one reason I've bounced off of this in the past is that IMO this is not an improvement. I think one line per dependency/features is a pretty strong default and having just a few dependencies that spread their features out over multiple lines does not make the `Cargo.toml` easier to read.

Reply:
> So one reason I've bounced off of this in the past is that IMO this is not an improvement. I think one line per dependency/features is a pretty strong default and having just a few dependencies that spread their features out over multiple lines does not make the `Cargo.toml` easier to read.



@djc Can we make max line lengths extra large, like 200?

Reply:
@djc In other words I don't care that much about the style per se, but rather that the formatting can be automated...

Reply:
@djc Special rule applied: `column_width = 160` for `dependencies` exclusively.

Reply:
@djc Sorry, I forgot `dev-` and `build-dependencies`, just fixed.